extra/alacritty.man

```diff
@@ -39,8 +39,8 @@ Increases the level of verbosity (the max level is \fB\-vvv\fR)
 Prints version information
 .SH "OPTIONS"
 .TP
-\fB\-\-class\fR <class>
-Defines the window class on Linux [default: Alacritty]
+\fB\-\-class\fR [ <name> | <name>,<class> ]
+Defines the window class hint on Linux [default: Alacritty]
```

Comment:
This default should be adjusted to show that both instance and general class are set to Alacritty.